### ISSUE TITLE
Embeddinator shuts down without detailed error message

### ISSUE THREAD

**Issue by realvictorprm** [mono/Embeddinator-4000#17]
Hello fellows,

Embeddinator fails with a not detailed error message. That's not right, simply adding a description to the exception would help a lot.

E.g. the error code:

```
Unbehandelte Ausnahme: System.NotImplementedException: Die Methode oder der Vorgang ist nicht implementiert.
   bei MonoEmbeddinator4000.Driver.CreateGenerator() in D:\Users\Victor\Documents\Embeddinator-4000\binder\Driver.cs:Zeile 156.
   bei MonoEmbeddinator4000.Driver.Process() in D:\Users\Victor\Documents\Embeddinator-4000\binder\Driver.cs:Zeile 52.
   bei MonoEmbeddinator4000.Driver.Run() in D:\Users\Victor\Documents\Embeddinator-4000\binder\Driver.cs:Zeile 220.
   bei MonoEmbeddinator4000.CLI.Main(String[] args) in D:\Users\Victor\Documents\Embeddinator-4000\binder\CLI.cs:Zeile 182
```

**Comment by realvictorprm** [comment]
Fixed with #18

**Comment by realvictorprm** [comment]
Okay I discussed with @tritao that we should use a general error message appearing before the actual error message comes. So this will be adjusted again and stay till then open.
